h/static/scripts/test/root-thread-test.js

```diff
@@ -224,6 +237,19 @@ describe('rootThread', function () {
 
       assert.isFalse(threadFilterFn(fakeBuildThread));
     });
+
+    it('generates a thread filter function for annotations, when all annotations are of type orphans', function () {
```

Comment:
This test has the same type mismatch as https://github.com/hypothesis/client/pull/52/files#r72415420 and I have the same comment about the naming of the test.